setup.py

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/python
```

Comment:
It would be better to use `/usr/bin/env python3`, as `python` does not exist on Ubuntu 20.04

Reply:
Done!